packages/eas-cli/.eslintrc.js

```diff
@@ -32,6 +32,14 @@ module.exports = {
         ],
       },
     ],
+    'no-restricted-syntax': [
+      'error',
+      {
+        selector: "NewExpression[callee.name='EasJsonAccessor']",
+        message:
+          "Don't use EasJsonAccessor class, use EasJsonAccessorWithCustomErrors class instead!",
+      },
+    ],
```

Comment:
@dsokal What do you think about it?

Reply:
![Screenshot 2022-11-18 at 16 27 17](https://user-images.githubusercontent.com/55145344/202740824-6d86d97f-28d7-45f7-a9ff-a94ef1637752.png)

Reply:
Looks good!